api/v1alpha1/doc.go

```diff
@@ -1,7 +1,7 @@
 // Unless explicitly stated otherwise all files in this repository are licensed
 // under the Apache License Version 2.0.
 // This product includes software developed at Datadog (https://www.datadoghq.com/).
-// Copyright 2016-2019 Datadog, Inc.
+// Copyright 2016-2020 Datadog, Inc.
```

Comment:
```suggestion
// Copyright 2016-2021 Datadog, Inc.
```
;)

Reply:
I was suggested in DataDog/datadog-agent#7332 to use `present` instead of `2021` in the copyright mentions.
It seems to be what we do for other repositories.
```suggestion
// Copyright 2016-present Datadog, Inc.
```

Reply:
I guess I will  complete the To Do now heh